patches/minecraft/net/minecraft/entity/monster/EntityZombieVillager.java.patch

```diff
@@ -8,7 +8,28 @@
      }
  
      public int func_190736_dl()
-@@ -85,6 +86,21 @@
+@@ -64,6 +65,7 @@
+     {
```

Comment:
Should I go ahead and do it in this pr, so it won't be as much of an annoyance in the future?